nodejs/values.yaml

```diff
@@ -23,8 +23,8 @@ livenessFailureThreshold: 3
 startupPath: '/health/liveness'
 startupDelay: 5
 startupTimeout: 3
-startupPeriod: 10
-startupFailureThreshold: 3
+startupPeriod: 30
```

Comment:
Ha looks like the original PR has a mistake.

startupDelay should be modified not period

but frontend apps can start quickly so i would probably just increase the failure threshold and leave the other values.

maybe bump failure to 15?